core/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/models/EwmaModel.java

```diff
@@ -107,11 +114,42 @@ public String getName() {
     };
 
     @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.field(MovAvgParser.MODEL.getPreferredName(), NAME_FIELD.getPreferredName());
+        builder.startObject(MovAvgParser.SETTINGS.getPreferredName());
+        builder.field("alpha", alpha);
+        builder.endObject();
+        return builder;
+    }
+
+    @Override
+    public MovAvgModel readFrom(StreamInput in) throws IOException {
+        return new EwmaModel(in.readDouble());
+    }
+
+    @Override
     public void writeTo(StreamOutput out) throws IOException {
         out.writeString(STREAM.getName());
```

Comment:
I took another look at MovAvgModelStreams, and although I'm not completely sure it look a lot like what NamedWritable is doing, so I was wondering if Stream could be replaced by it.